catalog/app/components/SearchResults/SearchResults.js

```diff
@@ -457,7 +457,12 @@ function FileHit({ showBucket, hit: { path, versions, bucket } }) {
     })
 
   return (
-    <Section>
+    <Section
+      data-testid="search-hit"
+      data-search-hit-type="file"
+      data-search-hit-bucket={bucket}
+      data-search-hit-path={path}
```

Comment:
Probably, we can use some decorators for components, destructuring `data-search-hit={handle}` to this

Reply:
that's probably worth exploring